src/pages/SendTokens/TokenInfoBlock.js

```diff
@@ -0,0 +1,72 @@
+import { Paper, Grid, Typography } from '@mui/material';
+import { styled } from '@mui/system';
+import tokenIcon from './tokens.svg';
+
+
+const SubText = styled(Typography)({
+  fontSize: 16,
+  lineHeight: '24px',
+  fontWeight: 700,
+  color: 'rgb(97,137,47)',
+  textAlign: 'center',
+  overflowWrap: 'break-word',
+  padding: '10px',
+});
+
+const AmountText = styled(Typography)({
+  fontSize: 24,
+  lineHeight: '24px',
+  fontWeight: 700,
+  color: 'rgb(97,137,47)',
+  textAlign: 'center',
+  overflowWrap: 'break-word',
+  paddingTop: '10px',
+  paddingBottom: '10px',
+});
+
+
+const SubWalletInfo = ({ subWalletName, subWalletTokens }) => {
+  return (
```

Comment:
Personally, I would prefer naming 'wallet' instead of 'subWallet', because even the current Wallet can be selected as a sender wallet, not only a managed one. But it is up to you.

Reply:
I've renamed it as `senderWallet` instead. Is that more appropriate?